### PR TITLE
[zh]Update content/zh/docs/reference/kubectl/kubectl.md.

### PR DESCRIPTION
**Pull request by yuandongx**

### CODE REVIEW

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 1]
```diff
@@ -137,15 +137,24 @@ kubectl [flags]
       TLS 使用的客户端密钥文件路径
       </td>
     </tr>
+    <tr>
+      <td colspan="2">--cloud-provider-gce-l7lb-src-cidrs cidrs&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: 130.211.0.0/22,35.191.0.0/16</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+        <!--CIDRs opened in GCE firewall for L7 LB traffic proxy & health checks-->
+        在 GCE 防火墙中打开 CIDR，以进行 L7 LB 流量代理和运行状况检查。
```
Comment (howieyuen) [inline content/zh/docs/reference/kubectl/kubectl.md:146]: 健康检查

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 2]
```diff
@@ -137,15 +137,24 @@ kubectl [flags]
...
       <td colspan="2">--cloud-provider-gce-lb-src-cidrs cidrs&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;默认值: 130.211.0.0/22,209.85.152.0/22,209.85.204.0/22,35.191.0.0/16</td>
     </tr>
     <tr>
       <td></td><td style="line-height: 130%; word-wrap: break-word;">
       <!--
-      CIDRs opened in GCE firewall for LB traffic proxy & health checks
+      CIDRs opened in GCE firewall for L4 LB traffic proxy & health checks
       -->
-      在 GCE 防火墙中打开 CIDR，以进行 LB 流量代理和运行状况检查。
+      在 GCE 防火墙中打开 CIDR，以进行 L4 LB 流量代理和运行状况检查。
```
Comment (howieyuen) [inline content/zh/docs/reference/kubectl/kubectl.md:157]: 同上

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 3]
```diff
@@ -312,6 +321,15 @@ kubectl [flags]
       -->
       如果存在，CLI 请求将使用此命名空间
       </td>
+    <tr>
+      <td colspan="2">--one-output</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+      <!--If true, only write logs to their native severity level (vs also writing to each lower severity level-->
+      如果为 true，则只将日志写入其本机严重性级别（而不是同时写入每个较低的严重级别）。
```
Comment (howieyuen) [inline content/zh/docs/reference/kubectl/kubectl.md:330]: 如果为 true，则只将日志写入初始严重级别

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 4]
```diff
@@ -504,20 +520,20 @@ kubectl [flags]
...
+* [kubectl debug](/docs/reference/generated/kubectl/kubectl-commands#debug)	 - Create debugging sessions for troubleshooting workloads and nodes
 * [kubectl delete](/docs/reference/generated/kubectl/kubectl-commands#delete)	 - Delete resources by filenames, stdin, resources and names, or by resources and label selector
 -->
 * [kubectl convert](/docs/reference/generated/kubectl/kubectl-commands#convert)	 - 在不同的 API 版本之间转换配置文件
 * [kubectl cordon](/docs/reference/generated/kubectl/kubectl-commands#cordon)	 - 标记节点为不可调度的
 * [kubectl cp](/docs/reference/generated/kubectl/kubectl-commands#cp)	 - 将文件和目录拷入/拷出容器。
-* [kubectl create](/docs/reference/generated/kubectl/kubectl-commands#create)	 - 通过文件或标准输入来创建资源
-* [kubectl delete](/docs/reference/generated/kubectl/kubectl-commands#delete)	 - 通过文件名、标准输入、资源和名字删除资源，或者通过资源和标签选择器来删除资源
-
+* [kubectl create](/docs/reference/generated/kubectl/kubectl-commands#create)	 - 通过文件或标准输入来创建资源。
```
Comment (howieyuen) [inline content/zh/docs/reference/kubectl/kubectl.md:534]: /zh/docs/reference/generated/kubectl/kubectl-commands#create
1. I see a lot en links here, please help fix them all. add ”/zh“ prefix in order to redirect zh pages
2. about punctuation, I prefer to keep context consistent, if add, try add them all, or else none is added.

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 5]
```diff
@@ -534,23 +550,13 @@ kubectl [flags]
...
--->
-* [kubectl get](/docs/reference/generated/kubectl/kubectl-commands#get)	 - 显示一个或者多个资源信息
-* [kubectl kustomize](/docs/reference/generated/kubectl/kubectl-commands#kustomize)	 - 从目录或远程 URL 中构建 kustomization
-* [kubectl label](/docs/reference/generated/kubectl/kubectl-commands#label)	 - 更新资源的标签
-* [kubectl logs](/docs/reference/generated/kubectl/kubectl-commands#logs)	 - 输出 pod 中某容器的日志
-* [kubectl options](/docs/reference/generated/kubectl/kubectl-commands#options)	 - 打印所有命令都支持的共有参数列表
-* [kubectl patch](/docs/reference/generated/kubectl/kubectl-commands#patch)	 - 基于策略性合并修补（Stategic Merge Patch）规则更新某资源中的字段
-
-<!--
+* [kubectl patch](/docs/reference/generated/kubectl/kubectl-commands#patch)	 - Update field(s) of a resource
```
Comment (howieyuen) [inline content/zh/docs/reference/kubectl/kubectl.md:559]: same as above

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 6]
```diff
@@ -564,17 +570,16 @@ kubectl [flags]
 * [kubectl replace](/docs/reference/generated/kubectl/kubectl-commands#replace)	 - 基于文件名或标准输入替换资源
 * [kubectl rollout](/docs/reference/generated/kubectl/kubectl-commands#rollout)	 - 管理资源的上线
 * [kubectl run](/docs/reference/generated/kubectl/kubectl-commands#run)	 - 在集群中使用指定镜像启动容器
-
 <!--
-* [kubectl scale](/docs/reference/generated/kubectl/kubectl-commands#scale)	 - Set a new size for a Deployment, ReplicaSet, Replication Controller, or Job
+* [kubectl scale](/docs/reference/generated/kubectl/kubectl-commands#scale)	 - Set a new size for a Deployment, ReplicaSet or Replication Controller
```
Comment (howieyuen) [inline content/zh/docs/reference/kubectl/kubectl.md:574]: same as above

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 7]
```diff
@@ -564,17 +570,16 @@ kubectl [flags]
...
 * [kubectl set](/docs/reference/generated/kubectl/kubectl-commands#set)	 - Set specific features on objects
 * [kubectl taint](/docs/reference/generated/kubectl/kubectl-commands#taint)	 - Update the taints on one or more nodes
 * [kubectl top](/docs/reference/generated/kubectl/kubectl-commands#top)	 - Display Resource (CPU/Memory/Storage) usage.
 * [kubectl uncordon](/docs/reference/generated/kubectl/kubectl-commands#uncordon)	 - Mark node as schedulable
 * [kubectl version](/docs/reference/generated/kubectl/kubectl-commands#version)	 - Print the client and server version information
 * [kubectl wait](/docs/reference/generated/kubectl/kubectl-commands#wait)	 - Experimental: Wait for a specific condition on one or many resources.
 -->
-* [kubectl scale](/docs/reference/generated/kubectl/kubectl-commands#scale)	 - 为一个 Deployment、ReplicaSet、ReplicationController 或 Job 设置一个新的规模尺寸值
+* [kubectl scale](/docs/reference/generated/kubectl/kubectl-commands#scale)	 - 为一个 Deployment、ReplicaSet 或 ReplicationController 设置一个新的规模尺寸值
```
Comment (howieyuen) [inline content/zh/docs/reference/kubectl/kubectl.md:582]: same as above

**Review by howieyuen — CHANGES_REQUESTED** [review]

File: content/zh/docs/reference/kubectl/kubectl.md (hunk 4 above)
Comment (yuandongx) [reply content/zh/docs/reference/kubectl/kubectl.md:534]: @howieyuen 
I do't find the chinese links about them.
They are redirected to anther site in English, not Chinese.
Can you look up Chinese links?
image

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 8]
```diff
@@ -137,15 +137,24 @@ kubectl [flags]
       TLS 使用的客户端密钥文件路径
       </td>
     </tr>
+    <tr>
+      <td colspan="2">--cloud-provider-gce-l7lb-src-cidrs cidrs&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: 130.211.0.0/22,35.191.0.0/16</td>
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/kubectl.md:141]: "Default:" 译

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 9]
```diff
@@ -137,15 +137,24 @@ kubectl [flags]
...
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+        <!--CIDRs opened in GCE firewall for L7 LB traffic proxy & health checks-->
+        在 GCE 防火墙中打开 CIDR，以进行 L7 LB 流量代理和健康检查。
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/kubectl.md:146]:
```suggestion
        在 GCE 防火墙中开放的 CIDR，用来进行 L7 LB 流量代理和健康检查。
```

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 10]
```diff
@@ -137,15 +137,24 @@ kubectl [flags]
...
       <td colspan="2">--cloud-provider-gce-lb-src-cidrs cidrs&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;默认值: 130.211.0.0/22,209.85.152.0/22,209.85.204.0/22,35.191.0.0/16</td>
     </tr>
     <tr>
       <td></td><td style="line-height: 130%; word-wrap: break-word;">
       <!--
-      CIDRs opened in GCE firewall for LB traffic proxy & health checks
+      CIDRs opened in GCE firewall for L4 LB traffic proxy & health checks
       -->
-      在 GCE 防火墙中打开 CIDR，以进行 LB 流量代理和运行状况检查。
+      在 GCE 防火墙中打开 CIDR，以进行 L4 LB 流量代理和健康检查。
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/kubectl.md:157]:
```suggestion
      在 GCE 防火墙中中开放的 CIDR，用来进行 L4 LB 流量代理和健康检查。
```

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 11]
```diff
@@ -312,6 +321,15 @@ kubectl [flags]
       -->
       如果存在，CLI 请求将使用此命名空间
       </td>
+    <tr>
+      <td colspan="2">--one-output</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+      <!--If true, only write logs to their native severity level (vs also writing to each lower severity level-->
+      如果为 true，则只将日志写入初始严重级别（而不是同时写入每个较低的严重级别）。
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/kubectl.md:330]:
```suggestion
      如果为 true，则只将日志写入初始严重级别（而不是同时写入所有较低的严重级别）。
```

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 12]
```diff
@@ -504,20 +520,20 @@ kubectl [flags]
...
 * [kubectl create](/docs/reference/generated/kubectl/kubectl-commands#create)	 - Create a resource from a file or from stdin.
+* [kubectl debug](/docs/reference/generated/kubectl/kubectl-commands#debug)	 - Create debugging sessions for troubleshooting workloads and nodes
 * [kubectl delete](/docs/reference/generated/kubectl/kubectl-commands#delete)	 - Delete resources by filenames, stdin, resources and names, or by resources and label selector
 -->
 * [kubectl convert](/docs/reference/generated/kubectl/kubectl-commands#convert)	 - 在不同的 API 版本之间转换配置文件
 * [kubectl cordon](/docs/reference/generated/kubectl/kubectl-commands#cordon)	 - 标记节点为不可调度的
-* [kubectl cp](/docs/reference/generated/kubectl/kubectl-commands#cp)	 - 将文件和目录拷入/拷出容器。
+* [kubectl cp](/docs/reference/generated/kubectl/kubectl-commands#cp)	 - 将文件和目录拷入/拷出容器
 * [kubectl create](/docs/reference/generated/kubectl/kubectl-commands#create)	 - 通过文件或标准输入来创建资源
+* [kubectl debug](/docs/reference/generated/kubectl/kubectl-commands#debug)	 - 创建用于排除工作负载和节点故障的调试会话
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/kubectl.md:535]:
```suggestion
* [kubectl debug](/docs/reference/generated/kubectl/kubectl-commands#debug)	 - 创建用于排查工作负载和节点故障的调试会话
```

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 13]
```diff
@@ -534,23 +550,13 @@ kubectl [flags]
...
 * [kubectl logs](/docs/reference/generated/kubectl/kubectl-commands#logs)	 - Print the logs for a container in a pod
 * [kubectl options](/docs/reference/generated/kubectl/kubectl-commands#options)	 - Print the list of flags inherited by all commands
-* [kubectl patch](/docs/reference/generated/kubectl/kubectl-commands#patch)	 - Update field(s) of a resource using strategic merge patch
--->
-* [kubectl get](/docs/reference/generated/kubectl/kubectl-commands#get)	 - 显示一个或者多个资源信息
-* [kubectl kustomize](/docs/reference/generated/kubectl/kubectl-commands#kustomize)	 - 从目录或远程 URL 中构建 kustomization
-* [kubectl label](/docs/reference/generated/kubectl/kubectl-commands#label)	 - 更新资源的标签
-* [kubectl logs](/docs/reference/generated/kubectl/kubectl-commands#logs)	 - 输出 pod 中某容器的日志
-* [kubectl options](/docs/reference/generated/kubectl/kubectl-commands#options)	 - 打印所有命令都支持的共有参数列表
-* [kubectl patch](/docs/reference/generated/kubectl/kubectl-commands#patch)	 - 基于策略性合并修补（Stategic Merge Patch）规则更新某资源中的字段
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/kubectl.md:551]: 这些内容呢？

**Review by tanjunchen — COMMENTED** [review]
/approve

**Pull request merged**